examples/lines_bars_and_markers/fill_demo.py

```diff
@@ -1,13 +1,13 @@
+import numpy as np
+import matplotlib.pyplot as plt
```

Comment:
Could the import lines be just below the title (as in the original file)? Otherwise this looks good if the tests pass!